howto/troubleshoot/bug-report.md

```diff
@@ -0,0 +1,21 @@
+*Applies to: Anbox Cloud, Anbox Cloud Appliance since 1.16.0*
+
+In some cases, you might encounter issues with Anbox Cloud that you are not able
+to troubleshoot yourself, or that might be bugs in the software itself. In this
+case, get in touch with the Anbox Cloud support team. They will ask you to
+create a bug report for your issue.
+
+Before you submit a bug report, check the instructions for
+commonly encountered problems at [How to troubleshoot Anbox
+Cloud](https://discourse.ubuntu.com/t/how-to-troubleshoot-anbox-cloud/17837).
+
+Anbox containers come preinstalled with the `anbox-bug-report` utility, which
+collects the log files and other relevant information for a specific container.
+To generate the report and save it to a local file, use `amc exec` on a running
+container:
+```
+amc exec <container_id> -- bash -c 'cat "$(anbox-bug-report)"' > "<target_file>"
+```
+
+This command builds a zip archive that contains the report. It then saves it to
+the local `<target_file>`. This process might take a few seconds.
```

Comment:
As this currently only describes how to collect information on the container level the site needs to grow to add other parts of the software stack too. We can leave that out but should record as a story in JIRA to improve in the near future.

For now I think we need to make clear that the steps described here to specific to bugs at the container level but eventually not sufficient for all bugs as it misses important information from other parts of the stack.

Reply:
For now, if we tell customers to only generate a bug report when asked to, it might not be necessary to be more explicit about what is happening. (See my other comment)

What do you think?

Reply:
I think we should still create different sections on this page to explain how to collects logs/information from the different parts of the stack. That way support or we on a bug report can explicitly link to the right set of steps to follow.

Reply:
I will add a subsection titled `How to generate a container bug report`, and add a small explanation.